docs/software/home_automation.md

```diff
@@ -116,6 +116,9 @@ Domoticz is a Home Automation System that lets you monitor and configure various
 
     - HTTP: `http://<your.IP>:8124`
     - HTTPS: `https://<your.IP>:8424`
+    - Username: `admin`
+    - Password: `domoticz`
+    - NB: We recommend to create a new user with admin privileges via **Setup** -> **Users** and remove the above default one, or at least change the default password.
```

Comment:
I always try to avoid these abbreviations and e.g. use "Remark:"

Reply:
Ah, or a "tip" box.

Reply:
Like so? This is the green box with the flame:
```suggestion

    !!! tip "Change default user and password"
        We recommend to create a new user with admin privileges via **Setup** -> **Users** and remove the default one, or at least change the default password.
```